scripts/scrub_mongo.py

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Why is this file removed? Obsolete?

Reply:
Yes, that's a very old file. We will keep this repo around as archive and reference so I saw no reason to move it.